src/scheduler/server_info.cpp

```diff
@@ -177,12 +176,11 @@ extern char **environ;
  *
  */
 server_info *
-query_server(status *pol, int pbs_sd)
+query_universe(status *pol, int pbs_sd)
```

Comment:
I was thinking, should we really rename it to query_universe? If we move this method to the server_info class then it should ideally be called query_server. Another reason was that we already have functions like update_universe_on_run that call update_server_on_run so query_universe may seem confusing to people.